hw/xbox/nv2a/nv2a.h

```diff
@@ -447,6 +447,14 @@ void *pfifo_puller_thread(void *opaque);
 void pgraph_destroy(PGRAPHState *pg);
 void update_irq(NV2AState *d);
 
+
+void pgraph_context_switch(NV2AState *d, unsigned int channel_id);
+void pgraph_wait_fifo_access(NV2AState *d);
+void pgraph_method(NV2AState *d,
+                   unsigned int subchannel,
+                   unsigned int method,
+                   uint32_t parameter);
+
```

Comment:
I don't think these have to be public?

Reply:
Due to splitting up nv2a.c into nv2a_pgraph.c and nv2a_pfifo.c